options/elf/include/elf.h

```diff
@@ -236,19 +236,43 @@ enum {
 	R_X86_64_TPOFF64 = 18,
 	R_X86_64_PC64 = 24,
 	R_X86_64_GOTPC32 = 26,
+	R_X86_64_TLSDESC = 36,
 	R_X86_64_IRELATIVE = 37,
 };
 
 enum {
+	R_386_NONE = 0,
+	R_386_32 = 1,
+	R_386_PC32 = 2,
+	R_386_COPY = 5,
+	R_386_GLOB_DAT = 6,
+	R_386_JMP_SLOT = 7,
+	R_386_RELATIVE = 8,
+	R_386_TLS_TPOFF = 14,
+	R_386_TLS_DTPMOD32 = 35,
+	R_386_TLS_DTPOFF32 = 36,
+	R_386_TLS_DESC = 41,
+	R_386_IRELATIVE = 42,
+};
+
+enum {
+	R_AARCH64_NONE = 0,
 	R_AARCH64_ABS64 = 257,
 	R_AARCH64_COPY = 1024,
 	R_AARCH64_GLOB_DAT = 1025,
 	R_AARCH64_JUMP_SLOT = 1026,
 	R_AARCH64_RELATIVE = 1027,
-	R_AARCH64_TLS_TPREL = 1030,
-	R_AARCH64_TLSDESC = 1031
+	R_AARCH64_TLS_DTPREL64 = 1028,
+	R_AARCH64_TLS_DTPMOD64 = 1029,
+	R_AARCH64_TLS_TPREL64 = 1030,
+	R_AARCH64_TLSDESC = 1031,
+	R_AARCH64_IRELATIVE = 1032,
 };
 
+#define R_AARCH64_TLS_DTPREL R_AARCH64_TLS_DTPREL64
+#define R_AARCH64_TLS_DTPMOD R_AARCH64_TLS_DTPMOD64
+#define R_AARCH64_TLS_TPREL R_AARCH64_TLS_TPREL64
```

Comment:
Why do these defines need to exist?

Reply:
They exist on Linux and some programs seem to expect these to be in place.